src/templates/error.js

```diff
@@ -1,22 +1,20 @@
 export default (id, message, errorCode, code) => {
-    const detail = code ? `(${errorCode}: ${code})` : '';
+    message += code ? ` (${errorCode}: ${code})` : '';
```

Comment:
Can you create a assign a new constant the new message value, instead of mutating `message`?

Reply:
Sure